notebooks/MAST/K2/K2_FFI/beginner_how_to_use_ffi.ipynb

```diff
@@ -0,0 +1,277 @@
3N6Ev7g3htHws9R8A8Apzr0lBrgIgGKF9AKA/MX95ocn82pJjvdk4fsBiQmDlSBHghJSEYOBrOa/zAvzTjAH5+/v77JPbxkUEgLblDm8KpQBAPLvdbtjv90UsDHY4HIbn5+esNcBcnl30rZReSZ9EbEsC+56lAK/RfwAAJCRXMU1Ni2e0cLyhcje5CwAASCpVw1RTYxbSZvP1H4Ah7WQ0E9+AGGIG9r11htps3pW1MEtp9ZTocDgMp9MpyzhpDOsL7BNSqnNZbzHdw8cP2UMoL5VYEwDU7vn5+afwTA77/X749OmTwAzBeHZBbKX1JaXVQ1sE9oHQ9B8AAAnIVUxTW/211QsNus1dAABAEjmaj5ZWh3uLpg6YYAxrxVwFfXyTrIlvZRCepCUxA/tQm5Kv72Ntpy956yjZGJy/v7+P/maacUwmrE8Mu91uGIYham9xvh3eVkPY4+HjB2+TBIDAdrvdsNvthu12m+TNl/v93tiMKJ6fn5M9u+gt7FVDrxBbycdgrE2vRCgWJ2aKkq+LtentWOo/AAAikKuYrtasxGZT37GGhgjtA1Cm06neAS7lyX0utdb05D6eQJUE6vtRcqATgGVqurZv3gnuXzMG6cfJbSEnuFlIiVTGiZrkUdvEYIEUWicgA+Qyvvlyt9sNj4+PwzCEW1jJQkqkcn4ex9wG/aipX9IrEcLSfsQCyMBc+g8AuiZXQUi5z6XWchUlc6whG6F9AMqlwSSEUs6h2pueUo4jAEWrKdQJU8ydbBZ7kplJbORQ47V9rFl4/7KX4fr7+/uv/v7aZLfzrx0XAgDaV1MA5aWHjx+EUWDQVwBxjOGW8zdgjub0Fj2+kRxoR639kl6JJdYsHqYnAdbSfwDQJbkKQijlHKo9VwFwhdA+hBRqAFPi4KPlfaNs4zlTSoNAXZw36zmGAECnSgvsQw41BvbPbd4J7s8heA9cUmv45CVvkgSA+MY3YI7mvK1SYAbq10rvMFft+y24zxRrgvojz1L6VPs1siSO5S/pP6BgLWcPWt43yiZXwRrOG4BkbnIXAM0IOYApbTDU8r5Rj9Np2h8omWsgAA2rPdgJ51IF9kNMcINYWrmut7IfADm1OBm4xX0CAADyaKW/aGU/COf93d1Xf9YS2AeAjrScPWh536iHXAUtcA0EGnabuwBoQozBwmZTxkC55X0D2rb2+nXpOrXms2u6BmqGAZhIIJKWpArsQ8lau65v3g3D6UvuKgDq1HJow5skAQAgrJb7h7e0ts/6pPblWkzYs5R+tXadzMmxBKrRcvag5X0D2iZXAZDUTe4CgAtyhyVzbx9gqaXXr6mrC65dhdD1FYCGtBbspG8C+9Dudb3V/QKIqYeJwD3sI23LFbgBAKDdfqLV/SKPz09PnqUAAGHknnebe/sAS8lV9MuCCJCN0D4AwDAsb0o0MwAATSg9sG9SGym0Hmxvff8AAK7RVwAAsfQW9G59f1vfP9LQf+BaEo5jCQBAUnIV9fO9gKyE9qF0uVYNsloRUKsl16+1TcmSr+/hOnu+cp7GD6BJwo+0ovTAPqTQyzW9l/0EWKunicA97SsAAMTQ25i6l/3tZT97MvdZyFKfn548R8E1JCDHEuCMXAXAPHIVfZLbgOyE9gGAduRoLNd8TukN5tx9EtIH6IrQI63IFdhPNTkOpujtmt7b/gLM1eNE4B73GQAAmK+33qG3/WW5MagvrA8AAFAIuYq0XuYocv4BshPahxDc1OZxvIBShL4etXh9u7RPGjyAbgk70opcgX0AgFIJZAAAAHPoIYBhsFAxX3NvCMexBKpkLuk8jhdQCrkKgGBucxcAzRgHFDFW99ls0g5YYq1QZNAFxDT32hXrmnQ6zasl9TV+idLrAwBYwAQy+FGvC7Fs3g3D6UvuKgD49N33V/9NqsnJDx8/TKoHSmERMgAgt96ChL3t70ivxFRjj6L36Fuv18oYHEuganIV15mXC8QkVwGQldA+hPbaACFWs1YyAyUAAGhSr+FOyDnJzAQ3WnEtJO8eA5BXqonAc8Me5//eZGUAAMjPuBzqk3Ih4/d3d55rdMi9IRzHEmiKXMWP5CoAALoitA+1sGIQQDixr6dzV4UDgEoIU9IKb6SEH6W4rs95m/35v01R2+bdvPoAWCfEmxnHz4g1edkbJAEA4LJeg4Qp93tKT5L6+6BXYq7xOYznK33o9d4Qg2MJMJFcBUA4chUAwQntA18zGAJqVPu1yy8QAaiAwD6tKCGwn/KNNpDL2jD8+PXuPwD1ixHs+PTd9yYxAwBAQj2Pv1Pt+5ze6fzf9vy9oXzv7+4E9xvm+hOOYwlQqNrnJgN9qv3aJVcBNEBoHwDoS6omzqpwAADFKSGwD6WIGYYP+fb605e4tW7eha0XoFaxJgbHfBNjrOC+N0hSAwuBQbncQ4DWCBHGt/bekWpRM71S3eY+7wjZcwjut8e9IRzHEgCArOQqAKIQ2oea5F4xyEAJKFEr16Xc13gAuMBbjmlBrYH9UuqAqWIE4MfPdD8CqEuKMEeqcArUTl8BaQgyAq0wxk4n1L1j/BzfO0I57yEsGsYwuL6E5FgCBJJ7zq1cBVCiVq5Lua/xACsJ7UMpSmjccm8fILbUzVsJ13YAWElAkhbUGtiHWGJd272xHoBRytCi4D5AHi0E1EPeP1o4HkCfjKUvi3l8Ytw7YvdHDx8/uOd1aHxmsia8//7uzrOXirg3hONYAixUwtzb3NsHiE2uAiAaoX0AAADgVQL7tEBgH9JIEdg/fXFvAoihhcnDgvsAzCWwD/TCOLlMMe8d+iNiWRveF9wvh2tEOI4lAAAA8NJN7gKAmWKtLHTtc1OvogQAAAArlRjYX/MmGiDO4gAWAgAIq4XgognXAG0T2Adgilh9gXsHtRO8BwCKIVcBAECBhPahJBq4Mmw2y/8Aac35uct1jZ2zXdcRAAoiwEjtSgzsQ6tiBOkBAHidhcBgHYF9AHrgHkVsnqkAANHJVZRBrgLqIVc
```

Comment:
```suggestion
    "![kepler_focal_plane_layout_channels_color.png](kepler_focal_plane_layout_channels_color.png)"
```
Is there some specific reason for the "attachment" part here?  If you look at this notebook you'll see that it now contains the data in this PNG image *inside* the notebook, which I don't think was intended, given that you've also checked the PNG into the repo.  If instead you leave off the `attachment` part, it just links to the file like a standard web page, which is also better for improving the load speed of the page.